website/projects/map4sci/src/app/map/map.component.ts

```diff
@@ -122,6 +114,8 @@ export class MapComponent {
   @Input() mapCenter: [number, number] = [0, 0];
   @Input() minimapOptions: MiniMapOptions = defaultMinimapOptions;
   @Input() mapMarkers: MapMarker[] = [];
+  @Input() edges: Edge[] | undefined = defaultEdges;
```

Comment:
@bherr2 I'm running in to an issue where datasets without configs are passing in their undefined value and it's overwriting the defaults I'm trying to set here.

Reply:
Did you try adding a getter that a la:
```typescript
getEdges(): Edge[] {
   return this.edges ?? defaultEdges;
}
```
In angular 12 the nullish coalesce can be used in the template so could be moved into there then.